src/group_theory/subgroup.lean

```diff
@@ -380,6 +380,10 @@ instance : has_bot (subgroup G) :=
 @[to_additive]
 instance : inhabited (subgroup G) := ⟨⊥⟩
 
+noncomputable instance fintype' [fintype G] : fintype (subgroup G) :=
```

Comment:
I think it makes sense to add this for `setlike` things in general.

Reply:
Ideally this would be computable too, which can be obtained by showing that subgroup is equivalent to a subtype I think

Reply:
It's true that it's computable, but not really useful I think since it's such an inefficient way of enumerating subgroups, it will be too slow on any group with more than about six elements. Also, to do that in general for set-like stuff you'd need to add decidability assumptions to the set-like class.

Reply:
I don't think you need decidability assumptions beyond `[decidable_eq G]` on this instance; the construction would be to just iterate all `finset G`s and check which ones satisfy `mul_mem`, `one_mem`, etc. Finset membership is decidable if equality is.